src/sentry/integrations/metric_alerts.py

```diff
@@ -25,6 +26,16 @@
     "percentage(users_crashed, users)": "% users crash free rate",
 }
 LOGO_URL = absolute_uri(get_asset_url("sentry", "images/sentry-email-avatar.png"))
+# These should be the same as the options in the frontend
+# COMPARISON_DELTA_OPTIONS
+TEXT_COMPARISON_DELTA = {
+    5: ("same time 5 minutes ago"),  # 5 minutes
+    15: ("same time 15 minutes ago"),  # 15 minutes
```

Comment:
Kinda the same thing here as the other PR - if we're saying "same time x minutes/hours ago" are we talking about granularity at the seconds/minutes scale? the day/week/month makes sense but this one is a bit confusing